docs/examples.md

```diff
@@ -109,21 +96,35 @@ _e("key"
 _e "key"
 ```
 
-Note1: we don't match the closing parenthesis as you may want to pass arguments to your translation function.
-
-Note2: the parser is smart about escaped single or double quotes you may have in your key.
+Please note that:
+- We don't match the closing parenthesis, as you might want to pass arguments to your translation function;
+- The parser is smart about escaped quotes (single or double) you may have in your key.
 
+### Command line
```

Comment:
This is not true anymore. The `-f` function doesn't exist anymore. You must use the config file.

Reply:
I removed this section.